jccli/version.py

```diff
@@ -8,5 +8,5 @@
 This module contains project version information.
 """
 
-__version__ = "0.0.10"  #: the working version
-__release__ = "0.0.10"  #: the release version
+__version__ = "0.0.11"  #: the working version
```

Comment:
did you already bump in a different PR?

Reply:
I did. I'm confused why this is showing up here